<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="kutuphane_cekme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Kütüphane Çek</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="siniflar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>355</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>355</width>
      <height>210</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>355</width>
      <height>210</height>
     </size>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="siniflar_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>353</width>
       <height>208</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resim_cek">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resimleri Çek</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="siniflar_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>355</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>355</width>
      <height>210</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>355</width>
      <height>210</height>
     </size>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="siniflar_widget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>353</width>
       <height>208</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="listeye_aktar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SEÇİLENLERİ LİSTEYE AKTAR</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="yukleme_miktari">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="tespit_yap">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TESPİT YAP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iptal_et">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>İPTAL ET</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>151</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="linkedin">
       <property name="text">
        <string>LİNKEDIN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="github">
       <property name="text">
        <string>GİTHUB</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>281</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="search_input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 118, 123);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="arama_button">
         <property name="text">
          <string>Ara</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="result_label">
       <property name="text">
        <string>İÇE AKTARILAN LİSTEDE ARA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QDoubleSpinBox" name="esik_deger">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_esik">
       <property name="text">
        <string>EŞİK DEĞER</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
